ingest/core/query_test.py

```
@@ -0,0 +1,209 @@
+Allows testing search functionality and exploring the vector database.
+"""
+
+import argparse
+import sys
+from typing import Any, Dict, List
+
+from ..services import PineconeVectorStore
+from .config_loader import load_config
+
+
+def print_search_results(results: List[Dict[str, Any]], query: str) -> None:
```

Ruff (0.13.1)

Unused function argument: `query`

(ARG001)